client/wfnews-war/src/main/angular/src/app/utils/index.ts

```diff
@@ -789,3 +789,51 @@ export function showPanel(panelClass: string) {
 
 
 export const isAndroidViaNavigator = () => navigator.platform.includes('Linux') || navigator.platform.includes('Android');
+
+export function displayItemTitle(identifyItem) {
+  switch (identifyItem.layerId) {
+    case 'active-wildfires-fire-of-note':
+      return 'Wildfire of Note';
+    case 'active-wildfires-out-of-control':
+    case 'active-wildfires-under-control':
+    case 'bcws-activefires-publicview-inactive':
+    case 'active-wildfires-holding':
+    case 'active-wildfires-out':
+      return 'Wildfire';
+  }
+}
+
+export function getDescription(code: string) {
```

Comment:
incident-info-panel has the same code, can you update that component and let it import from here instead too?

Reply:
Sure, done